package.json

```diff
@@ -122,7 +122,7 @@
     "rome": "^12.1.0"
   },
   "dependencies": {
-    "@autometrics/typescript-plugin": "^0.4.0",
+    "@autometrics/typescript-plugin": "latest",
```

Comment:
This seems to be something that could hurt us in the future. I know it's a bit of a hassle to do this but perhaps it's a good idea of creating a pr on the vscode extension as part of the release process for the autometrics-ts library?

Reply:
yeah that's a good point. Not sure how far we can automate it but I'll pin it to `0.5.0` for now and look into making it part of the release CI on `autometrics-ts`